Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1262,6 +1272,13 @@ extension BrowserViewController: BrowserDelegate {
         let sessionRestoreHelper = SessionRestoreHelper(browser: browser)
         sessionRestoreHelper.delegate = self
         browser.addHelper(sessionRestoreHelper, name: SessionRestoreHelper.name())
+
+        let openURL = {(url: NSURL) -> Void in
+            log.info("Opening \(url)")
```

Comment:
This log entry prints the URL to the log files stored on disk. We should not leak that. Please remove.